plugins/modules/rds_cluster.py

```diff
@@ -949,10 +986,13 @@ def get_restore_snapshot_options(params_dict):
         "DatabaseName",
         "EnableCloudwatchLogsExports",
         "EnableIAMDatabaseAuthentication",
+        "PerformanceInsightsEnabled",
```

Comment:
I just noticed the configuration name doesn't match the response name.

```suggestion
        "EnablePerformanceInsights",
```